salt/modules/boto_asg.py

```diff
@@ -141,7 +141,7 @@ def get_config(name, region=None, key=None, keyid=None, profile=None):
     retries = 30
     while True:
         try:
-            asg = [g for g in conn.get_all_groups(names=[name]) if g.name]
+            asg = [g for g in conn.get_all_groups(names=[name]) if all([g.name, g.vpc_zone_identifier])]
```

Comment:
do we still want this here? Or can we roll is back out?

Reply:
Yep, both of those versions of that line (-/+) have been reverted back to the original.